charts/egeria-platform/values.yaml

```diff
@@ -0,0 +1,117 @@
+# Egeria Settings
+egeria:
+  # Global Egeria Settings
+  logging: error
+  version: "3.5"
```

Comment:
suggest we use 3.4

Reply:
See https://github.com/odpi/egeria-charts/pull/107#discussion_r777954776